system/Test/ControllerTester.php

```diff
@@ -164,7 +164,10 @@ public function execute(string $method, ...$params)
 		}
 		catch (Throwable $e)
 		{
-			$result->response()->setStatusCode($e->getCode());
+			if ($code >= 100 || $code < 600)
+			{
+				$result->response()->setStatusCode($e->getCode());
+			}
```

Comment:
```suggestion
			$code = $e->getCode();

			if ($code >= 100 || $code < 600)
			{
				$result->response()->setStatusCode($code);
			}
```

Reply:
I realized this mistake lying in bed last night and thought to myself "I wonder if anyone will notice before I can fix it?" 😋

I pasted this from ResponseTrait. The OR needs to be AND as well. I had a computer science professor who insisted you should never paste code, maybe she was right...